solr/solrj/src/java/org/apache/solr/common/util/JavaBinCodec.java

```diff
@@ -1080,6 +1081,16 @@ public boolean writePrimitive(Object val) throws IOException {
       daos.writeByte(DATE);
       daos.writeLong(((Date) val).getTime());
       return true;
+    } else if (val instanceof Instant) {
+      writeStr(val.toString());
```

Comment:
do these all need a `return true` as well?

Reply:
From the context one would assume so..